parser/src/main/scala/aqua/parser/expr/ConstantExpr.scala

```diff
@@ -15,11 +15,16 @@ case class ConstantExpr[F[_]](
 
 object ConstantExpr extends Expr.Leaf {
 
-  override def p[F[_]: LiftParser: Comonad]: P[ConstantExpr[F]] = {
+  override def p[F[_]: LiftParser: Comonad]: P[ConstantExpr[F]] =
     ((((`const` *> ` ` *> Name
       .p[F] <* ` `) ~ `?`.?).with1 <* `=` <* ` `) ~ Value.`value`).map {
       case ((name, mark), value) =>
         ConstantExpr(name, value, mark.nonEmpty)
     }
-  }
+
+  def onlyLiteral[F[_]: LiftParser: Comonad]: P[(Name[F], Literal[F])] =
```

Comment:
Maybe return the same `ConstantExpr`? Just for convinience

Reply:
i cannot do this, cause i need `Literal` instead of top-level `Value`